### PR TITLE
More parallelism on windows generate

### PR DESCRIPTION
**Pull request by dhiltgen**
Make the build faster

### CODE REVIEW

File: llm/generate/gen_windows.ps1 [hunk 1]
```diff
@@ -121,10 +121,15 @@ function build {
     write-host "generating config with: cmake -S ${script:llamacppDir} -B $script:buildDir $script:cmakeDefs"
     & cmake --version
     & cmake -S "${script:llamacppDir}" -B $script:buildDir $script:cmakeDefs
-    if ($LASTEXITCODE -ne 0) { exit($LASTEXITCODE)}
-    write-host "building with: cmake --build $script:buildDir --config $script:config $($script:cmakeTargets | ForEach-Object { `"--target`", $_ })"
-    & cmake --build $script:buildDir --config $script:config ($script:cmakeTargets | ForEach-Object { "--target", $_ })
-    if ($LASTEXITCODE -ne 0) { exit($LASTEXITCODE)}
+    if ($LASTEXITCODE -ne 0) { throw($LASTEXITCODE)}
+    if ($cmakeDefs -contains "-G") {
+        $extra=@("-j8")
```
Comment (jmorganca) [inline llm/generate/gen_windows.ps1:126]: Do you know if a simple `-j` will auto select the number of processors like with `make`?

File: llm/generate/gen_windows.ps1 [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #!powershell
 
-$ErrorActionPreference = "Stop"
+# $ErrorActionPreference = "Stop"
```
Comment (jmorganca) [inline llm/generate/gen_windows.ps1:3]: Did you mean to comment this out?

**Review by jmorganca — APPROVED** [review]
LGTM, small comment on line 3

File: llm/generate/gen_windows.ps1 (hunk 1 above)
Comment (dhiltgen) [reply llm/generate/gen_windows.ps1:126]: I thought -j without argument was unbounded parallelism.

File: llm/generate/gen_windows.ps1 (hunk 2 above)
Comment (dhiltgen) [reply llm/generate/gen_windows.ps1:3]: I'll remove the line.  Start-ThreadJob has some quirks around how errors are interpreted (the biggie is any stderr output is considered and error, ugh). exit also doesn't work in the thread job, so I had to switch to throw to get error handling to work right and detect a failed job.

**Pull request merged**